streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -246,35 +262,38 @@ public synchronized void onChange(final StreamThread thread, final StreamThread.
             }
         }
     }
+
     /**
-     * Construct the stream instance.
+     * Create a {@code KafkaStreams} instance.
      *
      * @param builder the processor topology builder specifying the computational logic
-     * @param props   properties for the {@link StreamsConfig}
+     * @param props   properties for {@link StreamsConfig}
      */
     public KafkaStreams(final TopologyBuilder builder, final Properties props) {
         this(builder, new StreamsConfig(props), new DefaultKafkaClientSupplier());
     }
 
     /**
-     * Construct the stream instance.
+     * Create a {@code KafkaStreams} instance.
      *
      * @param builder the processor topology builder specifying the computational logic
-     * @param config  the stream configs
+     * @param config  the Kafka Streams configuration
      */
     public KafkaStreams(final TopologyBuilder builder, final StreamsConfig config) {
         this(builder, config, new DefaultKafkaClientSupplier());
     }
 
     /**
-     * Construct the stream instance.
+     * Create a {@code KafkaStreams} instance.
      *
      * @param builder        the processor topology builder specifying the computational logic
-     * @param config         the stream configs
-     * @param clientSupplier the kafka clients supplier which provides underlying producer and consumer clients
-     *                       for this {@link KafkaStreams} instance
+     * @param config         the Kafka Streams configuration
+     * @param clientSupplier the Kafka clients supplier which provides underlying producer and consumer clients
```

Comment:
Nor required, as a link is automatically created in the JavaDoc in the method name+parameters (very first generated line)
```
org.apache.kafka.streams

public KafkaStreams(TopologyBuilder builder,
                                    StreamsConfig config,
                                    KafkaClientSupplier clientSupplier) // this contains a link

Create a KafkaStreams instance.

Parameters:
          builder - the processor topology builder specifying the computational logic
          config - the Kafka Streams configuration
          clientSupplier - the Kafka clients supplier which provides underlying producer and consumer clients for the new {@code KafkaStreams} instance
```